File: app/src/main/java/de/xikolo/controller/SettingsActivity.java
package de.xikolo.controller;

import android.app.FragmentManager;
import android.app.FragmentTransaction;
import android.os.Bundle;

import de.xikolo.R;
import de.xikolo.controller.settings.SettingsFragment;

public class SettingsActivity extends BaseActivity {

    public static final String TAG = SettingsActivity.class.getSimpleName();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
        setupActionBar();

        setTitle(getString(R.string.title_section_settings));

        String tag = "settings";

        FragmentManager fragmentManager = getFragmentManager();
        if (fragmentManager.findFragmentByTag(tag) == null) {
            FragmentTransaction transaction = fragmentManager.beginTransaction();
            transaction.replace(R.id.content, SettingsFragment.newInstance(), tag);
            transaction.commit();
        }
    }

}

File: app/src/main/java/de/xikolo/controllers/settings/SettingsActivity.kt
package de.xikolo.controllers.settings

import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import de.xikolo.R
import de.xikolo.controllers.base.BaseActivity
import de.xikolo.controllers.dialogs.CreateTicketDialog
import de.xikolo.controllers.dialogs.CreateTicketDialogAutoBundle

class SettingsActivity : BaseActivity() {

    companion object {
        val TAG: String = SettingsActivity::class.java.simpleName
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_blank)
        setupActionBar()

        title = getString(R.string.title_section_settings)

        val tag = "settings"

        val fragmentManager = supportFragmentManager
        if (fragmentManager.findFragmentByTag(tag) == null) {
            val transaction = fragmentManager.beginTransaction()
            transaction.replace(R.id.content, SettingsFragment(), tag)
            transaction.commit()
        }
    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        menuInflater.inflate(R.menu.helpdesk, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            R.id.action_helpdesk -> {
                val dialog = CreateTicketDialogAutoBundle.builder().build()
                dialog.show(supportFragmentManager, CreateTicketDialog.TAG)
                return true
            }
        }
        return super.onOptionsItemSelected(item)
    }

}
